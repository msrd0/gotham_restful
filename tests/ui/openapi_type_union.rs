#[macro_use] extern crate gotham_restful;

#[derive(OpenapiType)]
union IntOrPointer
{
	int: u64,
	pointer: *mut String
}

fn main()
{
}
