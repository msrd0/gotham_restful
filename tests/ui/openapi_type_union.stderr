error: #[derive(OpenapiType)] only works for structs and enums
 --> $DIR/openapi_type_union.rs:4:1
  |
4 | union IntOrPointer
  | ^^^^^
